lightning/src/routing/network_graph.rs

```diff
@@ -1078,6 +1078,18 @@ impl NetworkGraph {
 		}
 	}
 
+	/// The unix timestamp in UTC provided by the most recent rapid gossip sync.
+	/// It will be set by the rapid sync process after every sync completion.
+	pub fn get_last_rapid_gossip_sync_timestamp(&self) -> Option<u32> {
```

Comment:
Do we describe anywhere how this should be used?

Reply:
Yeah, the rapid gossip sync crate docs talk about the timestamp